### PR TITLE
Make Gubernator pull logs for currently running builds from Prow.

### PR DESCRIPTION
**Pull request by rmmh**

### CODE REVIEW

File: gubernator/view_build.py [hunk 1]
```diff
@@ -54,6 +56,18 @@ def get_build_log(build_dir):
         return log_parser.digest(build_log)
 
 
+def get_running_build_log(job, build):
+    try:
+        url = "http://prow.k8s.io/log?pod=%s-%s" % (job, build)
```
Comment (spxtr) [inline gubernator/view_build.py]: Should you do the same check for the length that you do on prow's side?

Comment (rmmh) [inline gubernator/view_build.py]: done

**Comment by spxtr** [comment]
/lgtm

Nice! Please fix lint then feel free to deploy.

**Comment by ixdy** [comment]
Does this mean https://github.com/kubernetes/test-infra/issues/1376 is resolved?

**Pull request merged**